Source/cs505/group1/state/StateTester.java

```diff
@@ -0,0 +1,60 @@
+
+package cs505.group1.state;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Example of a Finite State Machine using the provided concrete subclass examples.
+ * 
+ * @author CS505 Fall17 Group1: <br>
+ * Emily Park<br>
+ * Jeffrey Blankenship  <br>
+ * Cecelia Oluwadoyinsola<br>
+ * James Luczynski<br>
+ * Melissa Mulcahy <br>
+ * @version 2017.10.29
+ * 
+ * To change the state by button operations:
+ * The line "ButtonStateA newState = new ButtonStateA();"
+ * in each method can be changed to a different state such as
+ * "ButtonStateA newState = new ButtonStateB();"
+ * This allows the method to return a different state.
+ */
+public class StateTester {

```

Comment:
Rename this to indicate it is instead a demo of how to use your component rather than just testing code.  If it is testing code then it should be in the testing directory instead.